frontend/src/metabase/dashboard/components/ClickBehaviorSidebar/ActionOptions/ActionOptions.tsx

```diff
@@ -96,19 +94,19 @@ function ActionOptions({
           />
         </ClickMappingsContainer>
       )}
-    </>
+    </ActionPickerWrapper>
   );
 }
 
 function ActionOptionsContainer(props: ActionOptionsProps) {
   return (
     <SidebarContent>
       <Heading className="text-medium">{t`Pick an action`}</Heading>
-      <Actions.ListLoader loadingAndErrorWrapper={false}>
-        {({ actions = [] }: { actions: WritebackAction[] }) => (
-          <ActionOptions {...props} actions={actions} />
-        )}
-      </Actions.ListLoader>
```

Comment:
we don't need to load all the actions anymore. The picker component handles fetching the actions only for the selected model